src/common/runner/runner.go

```diff
@@ -156,6 +158,15 @@ func (r *Runner) isSkippedResourceType(resourceType string) bool {
 	return false
 }
 
+func (r *Runner) isSkippedResource(resource string) bool {
+	for _, skippedResource := range r.skippedResources {
+		if resource == skippedResource {
+			return true
+		}
+	}
+	return false
+}
+
```

Comment:
@niradler You can import `github.com/bridgecrewio/yor/src/common/utils` and use the `InSlice` method instead